examples/distance-matrix/components/distance-matrix/distance-matrix.tsx

```diff
@@ -23,10 +23,12 @@ const DistanceMatrix = () => {
   const [destinationList, setDestinationList] = useState(['']);
 
   useEffect(() => {
-    if (!map || !service) {
-      return;
+    if (!map || !geocoder || !service) {
```

Comment:
Do we need to return here already, if we do not have a `geocoder`? Just thinking that we could still get the distance matrix response and do sth. with it in line 65, even if the `geocoder` is not working.

I know it might be a bit weird in this example, but might still be good to get the response and get some feedback for the geocoder  (hmmm)

Reply:
If we do so, the map and list would be out of sync. We would have results in the list, but no markers on the map. This is strange, isn't it?

Reply:
:D Yes, that is indeed strange. Just generally thought that through if it would make sense to return the results, even if we do not have a geocoder. But let's do it this way 👍